src/store/actions/predictions.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * I update predictions maximum size.
+ *
+ * @param {number} size - new predictions maximum size
+ *
+ * @returns {object} - action to be dispatched
+ */
+function setMaxSize(size) {
+    return {
+        type: 'SET_MAX_SIZE',
```

Comment:
Do you think there's benefit in centralizing the actions strings in a separate file as constants so we always use the same thing in actions and reducers?

for example:
actions/types.js
```
export SET_MAX_SIZE = 'SET_MAX_SIZE';
```

actions/predictions.js
```
import {SET_MAX_SIZE} from '../types';
...
type: SET_MAX_SIZE
...
```

reducers/predictions.js
```
import {SET_MAX_SIZE} from '../../actions/types';
...
case SET_MAX_SIZE:
...
```

Advantages I see: no errors caused by typos related to action names.
Disadvantages I see: more lines of code and a more complex file structure.